test/validate-reviews.test.js

```diff
@@ -0,0 +1,18 @@
+import app from "../server/server";
+import { describe, expect, test } from "@jest/globals";
+import validateReviews from "../server/validateReviews";                                                           
+
+describe("Test review validation", () => {
```

Comment:
This test suite doesn't really do anything, which is why it doesn't catch the fact that the validation logic is not working.

It contains one test that doesn't verify anything, and then a few more lines of code outside of the test. I think the mistake here is accidentally closing the testing function too early.